site2/docs/io-cdc-canal.md

```diff
@@ -1,174 +1,200 @@
+| `batchSize` | false | 1000 | Batch size to fetch from Canal. |
+
+### Example
+
+* JSON example
+
+    ```json
+    {
+        "zkServers": "127.0.0.1:2181",
+        "batchSize": "5120",
+        "destination": "example",
+        "username": "",
+        "password": "",
+        "cluster": false,
+        "singleHostname": "127.0.0.1",
+        "singlePort": "11111",
+    }
+    ```
+
+* [YAML example](https://github.com/apache/pulsar/blob/master/pulsar-io/canal/src/main/resources/canal-mysql-source-config.yaml)
+
+    ```yaml
+    configs:
+        zkServers: "127.0.0.1:2181"
+        batchSize: "5120"
+        destination: "example"
+        username: ""
+        password: ""
+        cluster: false
+        singleHostname: "127.0.0.1"
+        singlePort: "11111"
+    ```
+
+## Usage
+
+Here is an example of storing MySQL data using the configuration file as above.
+
+1. Start a MySQL server.
+
+    ```bash
+    $ docker pull mysql:5.7
+    $ docker run -d -it --rm --name pulsar-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=canal -e MYSQL_USER=mysqluser -e MYSQL_PASSWORD=mysqlpw mysql:5.7
+    ```
+
+2. Create a configuration file `mysqld.cnf`.
+
+    ```bash
+    [mysqld]
+    pid-file    = /var/run/mysqld/mysqld.pid
+    socket      = /var/run/mysqld/mysqld.sock
+    datadir     = /var/lib/mysql
+    #log-error  = /var/log/mysql/error.log
+    # By default we only accept connections from localhost
+    #bind-address   = 127.0.0.1
+    # Disabling symbolic-links is recommended to prevent assorted security risks
+    symbolic-links=0
+    log-bin=mysql-bin
+    binlog-format=ROW
+    server_id=1
+    ```
+
+3. Copy the configuration file `mysqld.cnf` to MySQL server.
+   
+    ```bash
+    $ docker cp mysqld.cnf pulsar-mysql:/etc/mysql/mysql.conf.d/
+    ```
+
+4.  Restart the MySQL server.
+   
+    ```bash
+    $ docker restart pulsar-mysql
+    ```
+
+5.  Create a test database in MySQL server.
+   
+    ```bash
+    $ docker exec -it pulsar-mysql /bin/bash
+    $ mysql -h 127.0.0.1 -uroot -pcanal -e 'create database test;'
+    ```
+
+6. Start a Canal server and connect to MySQL server.
+
+    ```
+    $ docker pull canal/canal-server:v1.1.2
+    $ docker run -d -it --link pulsar-mysql -e canal.auto.scan=false -e canal.destinations=test -e canal.instance.master.address=pulsar-mysql:3306 -e canal.instance.dbUsername=root -e canal.instance.dbPassword=canal -e canal.instance.connectionCharset=UTF-8 -e canal.instance.tsdb.enable=true -e canal.instance.gtidon=false --name=pulsar-canal-server -p 8000:8000 -p 2222:2222 -p 11111:11111 -p 11112:11112 -m 4096m canal/canal-server:v1.1.2
+    ```
+
+7. Start Pulsar standalone.
+
+    ```bash
+    $ docker pull apachepulsar/pulsar:2.3.0
+    $ docker run -d -it --link pulsar-canal-server -p 6650:6650 -p 8080:8080 -v $PWD/data:/pulsar/data --name pulsar-standalone apachepulsar/pulsar:2.3.0 bin/pulsar standalone
+    ```
+
+8. Start a Pulsar IO connector standalone.
+
```

Comment:
@tuteng Is line 118 correct?
If so, do we need a code example to explain more clearly?